app/views/openid_connect/authorizations/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2><%= @client.name %></h2>
+<p>You will be redirected to <%= @redirect_uri %> with an id token if approved or an error if denied</p>
+<ul>
+  <% @scopes.each do |scope| %>
+    <li><%= scope.name %></li>
+  <% end %>
+  <% if @request_object %>
+    <li>Request Objects (Currently not supported)</li>
+    <ul>
+      <pre><%= JSON.pretty_generate @request_object.as_json %></pre>
+    </ul>
+  <% end %>
+</ul>
+<%= render 'openid_connect/authorizations/form', action: :approve %>
+<%= render 'openid_connect/authorizations/form', action: :deny %>
```

Comment:
@AugierLe42e Yup, go ahead and modify as you wish I just added it as a stub. I'm not concentrating on the front-end/views for all these flows right now.